frontend/src/routes/(authenticated)/project/[project_code]/+page.svelte

```diff
@@ -379,6 +389,22 @@
           </AdminContent>
         </DetailItem>
       {/if}
+      <!-- Only show model version to project managers and admins, too technical to show to regular members -->
+      {#if project.type === ProjectType.FlEx}
```

Comment:
your comment doesn't match the condition, is this missing or is the comment no longer valid since it's the flex version?

Reply:
Removed the check, so I should remove the comment.

Reply:
Done.